### ISSUE TITLE
Splitter separator should have aria attributes if it is focusable (accessibility)

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-react#1274]
As described in the following link, elements with role "separator" that are focusable should have:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role#associated_wai-aria_roles_states_and_properties 
- aria-valuemax 
- aria-valuemin 
- aria-valuenow 

The separator element of the splitter is focusable and is with role "separator", so it should meet those requirements
